tools/IoTEdgeDevOps/DevOpsLib/JsonPathConverter.cs

```diff
@@ -28,6 +30,18 @@ class JsonPathConverter : JsonConverter
                     .FirstOrDefault();
 
                 string jsonPath = (att != null ? att.PropertyName : prop.Name);
+
+                if (serializer.ContractResolver is DefaultContractResolver)
+                {
+                    var resolver = (DefaultContractResolver) serializer.ContractResolver;
+                    jsonPath = resolver.GetResolvedPropertyName(jsonPath);
+                }
+
+                if (!Regex.IsMatch(jsonPath, @"^[a-zA-Z0-9_.-]+$"))
```

Comment:
This implies case insensitive. `context.json` vs `Context.json` vs `CoNtExt.JsoN`...

Reply:
I added the reference source in comment and remove this unnecessary checking.